core/server/src/main/webapp/general.jsp

```diff
@@ -114,6 +114,38 @@
       </div>
     </div>
   </div>
+  <!--  show inconsistent path  -->
+  <% if ((Integer) request.getAttribute("inconsistentPaths") != 0) { %>
+  <div class="row-fluid">
+    <div class="accordion span14" id="accordion4">
+      <div class="accordion-group">
+        <div class="accordion-heading">
+          <a class="accordion-toggle" data-toggle="collapse" data-parent="#accordion4" href="#data4">
+            <h4>Inconsistent Files</h4>
```

Comment:
need to restart the alluxio master server for the latest status.
We have two options:
1. We can add some information in the WebUI, tell the user "need to restart to get the latest information"
2.Refresh WebUI get lastest status.  Add a thread in the background real-time check, but this method requires frequent calls "FileSystemMaster.checkConsistency ()", the internal implementation of this function will affect the performance of alluxio. At the same time due to frequent internal lock INode, may lead to deadlock.

We can use the first way to prompt the user. Later to add more advanced refresh function.
what do you think?